Assets/_Laz/Scripts/Laz/LazModelBehavior.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Laz
+{
+    public class LazModelBehavior : MonoBehaviour
```

Comment:
I recommend placing this script into the parent gameobject (Whereever LazMovementController is). 

We want the mesh to be a separate thing from the logic.

Reply:
Yeah good point